packages/orchestrator-ui-components/src/components/WfoForms/formFields/deprecated/IpNetworkField.tsx

```diff
@@ -48,17 +45,16 @@ function IpNetwork({
 }: IPvAnyNetworkFieldProps) {
     const { formRowStyle } = useWithOrchestratorTheme(getCommonFormFieldStyles);
 
-    const t = useTranslations('pydanticForms');
-    const [selectedPrefix, setSelectedPrefix] = useState<IpBlock | undefined>(
-        undefined,
-    );
-    const [manualOverride, setManualOverride] = useState(false);
```

Comment:
A lot of functionality is removed here and also from l.76 onwards. Are we sure that this not needed anymore. Anything form related is a bit of a rabbit hole.

Reply:
Yes, it still works as expected without any issues.